cl/visualizations/views.py

```diff
@@ -32,10 +32,14 @@ def render_visualization_page(request, pk, embed):
     status = None
     if viz.deleted:
         status = statuses.HTTP_410_GONE
+        title = u"Visualization deleted by owner"
     else:
         if viz.published is False and viz.user != request.user:
             # Not deleted, private and not the owner
             status = statuses.HTTP_401_UNAUTHORIZED
+            title = u"Private visualization"
```

Comment:
Not sure this matters, but do we want to let people know that it exists in the title?

Reply:
Should be OK. They only have a URL with an ID in it. The most you can learn is how many visualizations have been created. Doesn't say by whom or anything like that.